x-pack/solutions/security/test/cloud_security_posture_api/routes/graph.ts

```diff
@@ -694,6 +697,247 @@ export default function (providerContext: FtrProviderContext) {
         expect(response.body).to.have.property('edges').length(2);
         expect(response.body).not.to.have.property('messages');
       });
+
+      describe('Enrich graph with entity metadata', () => {
+        const enrichPolicyCreationTimeout = 10000;
+
+        before(async () => {
+          await kibanaServer.uiSettings.update({ 'securitySolution:enableAssetInventory': true });
+          await esArchiver.load(
```

Comment:
Since entity_store is an internal index, it is advised to load documents and delete with query. see example https://github.com/elastic/kibana/pull/228294/files#diff-3e6adc81145e4da2be0dd86e047da38ee2c49fbceb57a47d9dff9e69d7e11371

pay attention load docs (docsOnly) is relevant on serverless tests only